pkg/controller/certificates/requestmanager/requestmanager_controller.go

```diff
@@ -398,7 +395,7 @@ func (c *controller) createNewCertificateRequest(ctx context.Context, crt *cmapi
 		},
 	}
 
-	cr, err = c.createOrApply(ctx, cr)
+	cr, err = c.client.CertmanagerV1().CertificateRequests(cr.Namespace).Create(ctx, cr, metav1.CreateOptions{})
```

Comment:
Note: so apply is not needed here as this controller only ever creates a whole new CR object, so clashes with other controllers are not possible

Reply:
For CRDs, for whatever reason, a new object can only be created using the `Create` call. Attempting an Apply will result in a "object does not exist" error. I do not know what it is possible to create via Apply on core resources such as Secrets, but not with CRDs.